src/app/components/Image/index.tsx

```diff
@@ -13,7 +13,7 @@ import styles from './index.styles';
 import { RequestContext } from '../../contexts/RequestContext';
 import { FRONT_PAGE } from '../../routes/utils/pageTypes';
 
-interface Props {
+type Props = {
   alt: string;
   aspectRatio?: [x: number, y: number];
   attribution?: string;
```

Comment:
These were the lines I was referring to (sorry...github n00b) 😆 

Curious to know why you have changed from interface to type here if you wouldn't mind explaining?! @karinathomasbbc and I have been discussing interface vs type during some previous work (we are largely using interface) but we are still unsure on when to use which. 🤔